### PR TITLE
Release: 2.0.5

### PR DESCRIPTION
**Pull request by hutchic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +44,96 @@
...
+##### Plugins
+
+- Go: fix issue where the go-pluginserver would not reload Go plugins'
+  configurations.
+  Thanks [wopol](https://github.com/wopol) for the patch!
+  [#5866](https://github.com/Kong/kong/pull/5866)
+- basic-auth: avoid fetching credentials when password is not given.
+  Thanks [Abhishekvrshny](https://github.com/Abhishekvrshny) for the patch!
+  [#5880](https://github.com/Kong/kong/pull/5880)
+- cors: avoid overwriting upstream response `Very` header; new values are now
```
Comment (Tieske) [inline CHANGELOG.md:129]: Very = Vary

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,6 +44,96 @@
 - [0.9.9 and prior](#099---20170202)
 
 
+## [2.0.5]
+
+> Released 2020/06/30
+
+### Dependencies
+
+- Bump OpenSSL version from `1.1.1f` to `1.1.1.g`.
```
Comment (dndx) [inline CHANGELOG.md:53]:
```suggestion
- Bump OpenSSL version from `1.1.1f` to `1.1.1g`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -43,6 +44,96 @@
...
+  [#5820](https://github.com/Kong/kong/pull/5810)
+- Bump [go-pluginserver](https://github.com/Kong/go-pluginserver) from version
+  from `0.2.0` to `0.3.2`, leveraging [go-pdk](https://github.com/Kong/go-pdk) `0.3.1`.
+  See go-pdk changelog [here](https://github.com/Kong/go-pdk/blob/master/CHANGELOG.md#v031).
```
Comment (dndx) [inline CHANGELOG.md:57]:
```suggestion
  See [go-pdk changelog](https://github.com/Kong/go-pdk/blob/master/CHANGELOG.md#v031).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -43,6 +44,96 @@
...
+
+### Fixes
+
+##### Core
+
+- Fix a race condition leading to random TLS breakages in DB-less mode.
```
Comment (dndx) [inline CHANGELOG.md:63]:
```suggestion
- Fix a race condition leading to random config fetching failure in DB-less mode.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -43,6 +44,96 @@
...
+  [#5833](https://github.com/Kong/kong/pull/5833)
+- Fix issue where a respawned worker would not get the existing configuration
```
Comment (dndx) [inline CHANGELOG.md:65]:
```suggestion
- Fix issue where a respawned worker would not use the existing configuration
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -43,6 +44,96 @@
...
+  [#5943](https://github.com/Kong/kong/pull/5943)
+- Fix issue where providing a declarative configuration file containing
+  fields with explicit null values would result in an error.
+  [#5999](https://github.com/Kong/kong/pull/5999)
+- Fix issue where the balancer wouldn't be built for all workers.
+  [#5931](https://github.com/Kong/kong/pull/5931)
+- Fix issue where a declarative configuration file with primary keys specified
+  as numbers would result in an error.
+  [#6005](https://github.com/Kong/kong/pull/6005)
+- Fix
```
Comment (dndx) [inline CHANGELOG.md:96]:
```suggestion
```

**Comment by gszr** [comment]
Thanks @Tieske and @dndx, I've also replicated the fixes in `release/2.1.0`.

**Pull request merged**